src/scheduler/fifo.c

```diff
@@ -1561,34 +1561,9 @@ run_update_resresv(status *policy, int pbs_sd, server_info *sinfo,
 			update_job_comment(pbs_sd, array, buf);
```

Comment:
The behavior of synchronous jobs are just like you have explained. Even in asynchronous jobs the comment is updated for both normal jobs and array jobs. So if the first job fails the comment will be updated accordingly, replacing the 'begun' comment. For consistency i will move this block to server and place it along with the block where we are resetting the comment for normal jobs.